src/atoms/Button/Button.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import styled from 'styled-components';
+
+import { Colors } from 'context';
+const { mainColor, secondaryColor, mainText } = Colors;
+
+export interface ButtonProps {
```

Comment:
export interface IStyledButtonProps { // use it as an interface for StyledButton
  btnColor?: string;
}

export interface ButtonProps  extends IStyledButtonProps {
  text: string;
  onClick?: () => void;
}